package.json

```diff
@@ -19,11 +23,23 @@
     "@react-native-community/eslint-config": "^0.0.5",
     "babel-jest": "^24.9.0",
     "eslint": "^6.8.0",
+    "husky": ">=1",
     "jest": "^24.9.0",
     "metro-react-native-babel-preset": "^0.57.0",
     "react-test-renderer": "16.9.0"
   },
   "jest": {
     "preset": "react-native"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix",
```

Comment:
@anjumpaul the lint-staged setup currently aborts commit only if there is at least one error. It will continue to commit if there are warnings but no errors. Set it up such that lint-staged will halt at warnings also.

Reply:
Added the requested change